test/test-handle-promise-rejections.js

```diff
@@ -0,0 +1,82 @@
+var common = require('./common');
+var airbrake = require(common.dir.root).createClient(common.projectId, common.key);
+var assert = require('assert');
+var sinon = require('sinon');
+
+(function testNotifyForUnhandlePromiseRejections() {
+  sinon.stub(process, 'on');
+  sinon.stub(airbrake, 'notify');
+  sinon.stub(airbrake, 'log');
+
+  airbrake.handlePromiseRejections();
+
+  var event = process.on.args[0][0];
+  var handler = process.on.args[0][1];
+
+  assert.equal(event, 'unhandledRejection');
+
+  assert.ok(!airbrake.notify.called);
```

Comment:
Let's stick to one style. If we use this style, then line 76 needs a modification. If not, then this line needs to be changed to `assert.equal(airbrake.notify.called, false);`. I'm fine with either of those as long as we're consistent.